src/sparql/QueryRepresentation.ts

```diff
@@ -9,4 +9,5 @@ export type Condition = {
 export default interface QueryRepresentation {
 	conditions: Condition[];
 	limit?: number;
+	useLimit: boolean;
```

Comment:
We might need the `useLimit` flag in the store (not sure yet), but here in the `QueryRepresentation`, we can just set `limit` to `null` or omit it, if we do not want to have a limit there.

Reply:
I added it because there is a default of 100. In that case the limit is true since the start.

Reply:
But isn't that correct? My Idea would be something like the following:

<table>
<tr>
	<th>State</th>
	<th>`QueryRepresentation`</th>
<tr>
	<td>`limit = 100; useLimit = true` (the default)</td>
	<td>`limit = 100`</td>
<tr>
	<td>`limit = 100; useLimit = false` (checkbox unchecked)</td>
	<td>`limit = null`</td>
<tr>
	<td>`limit = 30; useLimit = true` (checkbox checked and value changed)</td>
	<td>`limit = 30`</td>
<tr>
	<td>`limit = 30; useLimit = false` (checkbox unchecked and value changed)</td>
	<td>`limit = 30`</td>
</table>

Reply:
ah, now I understand. Yes. That's true. Thanks for explaining.